README.md

```diff
@@ -28,94 +62,470 @@ Run the installation command:
+[`login`](#affinidi-login)
 
+
+### Create or activate a project:
+The `create` command creates and activates a project. Follow the prompts to choose a name or add a name directly after the command.
 ```
 affinidi create project
 ```
 
+The `use` command activates an already existing project:
+```
+affinidi use project [<project-id>]
+```
+
+You can also simply type this and follow the prompts to choose from a list of existing projects: 
+```
+affinidi use project
+```
+Full reference for each command can be found here:    
+[`create`](#affinidi-create)   
+[`use`](#affinidi-use)
+
+### Schemas
+To issue a VC you first have to create a schema or choose an existing one:
+
+1. A schema can be created using the `create` command. You will need to provide a *schemaName* and a *description* in text format as well as a *source* with the path to the json file with the schema properties.
+
+```
+ $ affinidi create schema [schemaName] --description=<value> --source=<value>
+```
+
+
+2. To see available schemas:
+
+```
+ $ affinidi list schemas
+```
+
+You will need the value of the property `jsonSchemaUrl` of the created or chosen schema to issue VCs.
+
+Please see [About Schemas and Verifiable Credentials](#about-schemas-and-verifiable-credentials) for a detailed explanation on schema structures and how to create and find schemas using the [Schema Manager](https://affinidi-schema-manager.prod.affinity-project.org/api-docs/#).   
+Full reference for each command can be found here:    
+[`create`](#affinidi-create)    
+[`list`](#affinidi-list)
+
+
+### Verifiable Credentials
+You can issue and verify VCs within Affinidi CLI.
+
+1. To issue a VC you need to provide the *email* of the VC's owner, the `jsonSchemaUrl` of the *schema* on which the VC is based, and a path to the json file with credential *data*.
+
+```
+$ affinidi issue-vc [EMAIL] --schema=<value> --data <value>
+```
+
+Please see [How to structure a JSON file to issue a VC](#how-to-structure-a-json-file-to-issue-a-vc) for more details.
+
+2. To verify a VC you need to provide a path to the json file with the credential *data* to be verified.
+
+```
+$ affinidi verify-vc--data=<value> 
+```
+Full reference for each command can be found here:    
+[issue-vc](#affinidi-issue-vc)  
+[verify-vc](#affinidi-verify-vc)  
+
 ### Generate an application
+The [Affinidi Reference App](https://github.com/affinidi/elements-reference-app-frontend) provides a simple web interface for issuing, claiming and verifying VCs. It can be quickly generated by typing: 
 
 ```
-affinidi generate-application -n "My application"
+$ affinidi generate-application
+```
+You can also specify a name with:
 ```
+$ affinidi generate-application --name=<value>
+```
+
+For the full reference, please see the [`affinidi generate-application`](#affinidi-generate-application) command below.
+
+&nbsp;
 
-### Issue a VC for an example schema
+&nbsp;
 
-- create a file vc.json:
+#
+## CLI Commands 
+### **autocomplete**
```

Comment:
It is a very long ReadMe file, and the section breaks with thin rules ( `#` )  are there to separate thematically grouped content, they are not compulsory but they probably help in navigating through the extended document

the `###` doesn't add emphasis, just a font-size change: I am only adding bold emphasis to the `###` headingst in the CLI Commands section, which works like a glossary/dictionary: having bold on glossary terms is a frequent solution: I want to avoid the look of an endless collection of lines with no visual hints for when each single unit of meaning starts and end (each term of the glossary) --- are there other more standard solutions to achieve a similar visual effect?